staging/src/k8s.io/apiserver/pkg/storage/feature/feature_support_checker.go

```diff
@@ -43,84 +46,95 @@ type FeatureSupportChecker interface {
 	// Supports check if the feature is supported or not by checking internal cache.
 	// By default all calls to this function before calling CheckClient returns false.
 	// Returns true if all endpoints in etcd clients are supporting the feature.
-	Supports(feature storage.Feature) (bool, error)
+	Supports(feature storage.Feature) bool
 	// CheckClient works with etcd client to recalcualte feature support and cache it internally.
 	// All etcd clients should support feature to cause `Supports` return true.
 	// If client A supports and client B doesn't support the feature, the `Supports` will
 	// first return true at client A initializtion and then return false on client B
 	// initialzation, it can flip the support at runtime.
-	CheckClient(ctx context.Context, c client, feature storage.Feature) error
+	CheckClient(ctx context.Context, c client, feature storage.Feature)
```

Comment:
Regarding Lukasz' comment, the last three lines of this comment should also be copied to the comment of Supports() method.

Reply:
Done.